metrics-proxy/src/main/java/ai/vespa/metricsproxy/metric/model/DimensionId.java

```diff
@@ -1,17 +1,23 @@
 // Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package ai.vespa.metricsproxy.metric.model;
 
+import com.yahoo.yolean.concurrent.CopyOnWriteHashMap;
```

Comment:
Seems like there is a copy of the same class in vespajlib that could be used here. I would like to remove the yolean module and fold its unique contents into vespajlib.